mantidproject/mantid
# Indirect JumpFit - Scale

The Indirect JumpFit Interface scales FWHM to HWHM. A separate HWHM workspace is now created when data is loaded into the interface to prevent the original data being modified.

**To test:**

Input Files: 
[irs26176_graphite002_conv_1LFixF_s0_to_9_Result.zip](https://github.com/mantidproject/mantid/files/729154/irs26176_graphite002_conv_1LFixF_s0_to_9_Result.zip)

Load data into Mantid as a workspace.
Open Interfaces>Indirect>DataAnalysis>JumpFit
Load and re-load the workspace into the interface a few times (a quick way to do this is to switch between `File` and `Workspace` for the sample input)
Ensure the original input workspace is not modified.
Check `_HWHM` workspace is a 0.5 scale of the original input workspace.
The plot preview pane should also display the `_HWHM` workspace.


Fixes #18269.

[Release Notes](https://github.com/mantidproject/mantid/commit/1bf5f249ea6ec75116a617ecc91cac0a57ca7f7d)

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
